dpmData.py

```diff
@@ -1,47 +1,83 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """
-Created on Mon Apr 20 11:43:17 2020
+Created on Tue Jun 16 13:20:08 2020
 
 @author: snag
 """
```

Comment:
> I ran `python dpmData.py --help` and it simply ran the program rather than printing the help screen. Why isn't argparse working in this way?



> I ran `python dpmData.py --help` and it simply ran the program rather than printing the help screen. Why isn't argparse working in this way?

Let me look into this